index.bs

```diff
@@ -131,28 +131,25 @@ The IntersectionObserverCallback</h3>
 </pre>
 
 This callback will be invoked when there are changes to <a>target</a>'s
-intersection with <a>root</a>, as per the <a>processing model</a>.
+intersection with the <a>intersection root</a>, as per the
+<a>processing model</a>.
 
 <h3 id='intersection-observer-interface'>
 The IntersectionObserver interface</h3>
 
-The {{IntersectionObserver}} interface can be used
-to observe changes in the intersection of a <a>target</a> {{Element}}
-and a <a>root</a> {{Element}}
-(or the top-level document's viewport).
-
-Note: In {{MutationObserver}},
-the {{MutationObserverInit}} options are passed to {{MutationObserver/observe()}}
-while in {{IntersectionObserver}} they are passed to the constructor.
-This is because for MutationObserver,
-each {{Node}} being observed could have a different set of attributes to filter.
-For {{IntersectionObserver}},
-having different {{IntersectionObserverInit/root}},
-{{IntersectionObserverInit/rootMargin}}
-or {{threshold}} values
-for each <a>target</a>
-seems to introduce more complexity without solving additional use-cases.
-Per-{{observe()}} options could be provided in the future if V2 introduces a need for it.
+The {{IntersectionObserver}} interface can be used to observe changes in the
+intersection of a <a>target</a> {{Element}} and a {{IntersectionObserver/root}}
+{{Element}} (or the top-level document's viewport).
+
+Note: In {{MutationObserver}}, the {{MutationObserverInit}} options are passed
```

Comment:
I think the consensus is:

For typical usage, developers should feel free to create one observer per target.

In the rare case where many targets are observing the same root with the same parameters; and there is a likelihood that multiple notifications will be generated during a single run of the algorithm; then the grouping of notifications in the callback may offer a performance advantage.